lib/components/molecules/Accordian/tests/Accordian.test.js

```diff
@@ -0,0 +1,15 @@
+import { shallow } from 'enzyme';
+// import renderer from 'react-test-renderer';
+
+import Accordian from '../index';
+
+describe('<Accordian />', () => {
```

Comment:
The test needs to be much better, you need to test the expend collapse functionality